File: features/stacks/typescript/feature-code.tpl.ts
export type FeatureContext = {
  userId?: string;
  requestId: string;
};

export class FeatureImplementation {
  // Generated stub for feature: [[FEATURE_ID]]
  execute(_ctx: FeatureContext, _inputs: Record<string, unknown>): Record<string, unknown> {
    throw new Error('TODO: implement [[FEATURE_ID]]');
  }
}

File: features/stacks/go/feature-code.tpl.go
package features

import "fmt"

// Generated stub for feature: [[FEATURE_ID]]

type FeatureContext struct {
	UserID    string
	RequestID string
}

type FeatureImplementation struct{}

func (f *FeatureImplementation) Execute(ctx FeatureContext, inputs map[string]any) (map[string]any, error) {
	_ = ctx
	_ = inputs
	return nil, fmt.Errorf("TODO: implement [[FEATURE_ID]]")
}
